qiskit/transpiler/basepasses.py

```diff
@@ -52,20 +49,13 @@ def __init__(self):
         self.requires = []  # List of passes that requires
         self.preserves = []  # List of passes that preserves
         self.property_set = PropertySet()  # This pass's pointer to the pass manager's property set.
+        self._hash = None
 
-    @classmethod
-    def normalize_parameters(cls, *args, **kwargs):
```

Comment:
By removing this functionality you are removing a way of normalizing passes. Imagine a pass depending on an angle, without a normalization stage, these two passes:

```python
pass_a = AngledPass(math.pi)
pass_b = AngledPass(-math.pi)
``` 

Would be different and they are not.

I'm not blocking on this. If you think it is not needed, go for it. I just want to warn you.

Reply:
Fair warning. I don't think it would be a problem for now.